### PR TITLE
add please program

### PR DESCRIPTION
**Pull request by vahidnameni**
ADD please, sudo like program with regex support written in rust

changelog:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,9 +81,13 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
...
 * [volta](https://github.com/volta-cli/volta) - Volta: JS Toolchains as Code. ⚡
 
+#### please
+
+* [please](https://github.com/dalance/procs) - `sudo` like program with regex support written in rust
+
 #### ps
 
-* [procs](https://github.com/dalance/procs) - A modern replacement for ps written in Rust
+* [procs](https://gitlab.com/edneville/please) - A modern replacement for ps written in Rust
```
Comment (TaKO8Ki) [inline README.md]: I think you changed the wrong line by mistake.

```suggestion
* [procs](https://github.com/dalance/procs) - A modern replacement for ps written in Rust
```

File: README.md [hunk 2]
```diff
@@ -81,9 +81,13 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
 * [fnm](https://github.com/Schniz/fnm) - 🚀 Fast and simple Node.js version manager, built in Rust
 * [volta](https://github.com/volta-cli/volta) - Volta: JS Toolchains as Code. ⚡
 
+#### please
+
+* [please](https://github.com/dalance/procs) - `sudo` like program with regex support written in rust
```
Comment (TaKO8Ki) [inline README.md]:
```suggestion
#### sudo

* [please](https://gitlab.com/edneville/please) - `sudo` like program with regex support written in rust
```

**Comment by vahidnameni** [comment]
Yeah 😅!
Do I need to resend a new request?

**Comment by TaKO8Ki** [comment]
@vahidnameni 
No need for that. Could you accept my two commit suggestions in this PR?

**Comment by vahidnameni** [comment]
If you want to override, Yes I confirm two suggestion otherwise if you want i can commit suggestion from here.
Regards

**Review by TaKO8Ki — APPROVED** [review]
Thank you :)

**Pull request merged**